### PR TITLE
Version 1.8.0

### PR DESCRIPTION
**Pull request by shamsimam**
Please 
- do not squash during merge
- choose the rebase and merge option
- excuse the name of the branch.

### CODE REVIEW

File: executor/cook/_version.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # This file is read by setup.py to obtain the version.
 # Be aware that changing the format may break the parsing logic.
 
-__version__ = "0.1.2.dev0"
+__version__ = "0.1.3.dev0"
```
Comment (dposada) [inline executor/cook/_version.py]: What does the "dev0" mean? Do we need it?

File: scheduler/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.7.1] - 2017-11-07
+### Added
+- Added support for killing a group of jobs, from @DaoWen
+- Added sysouts to get job output closer to Mesos' CommandExecutor, from @shamsimam
+- Added metrics support for usage of /list, from @dposada
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Nitpick: I think we should remove "support"

File: scheduler/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
...
+- Added support for groups on retry endpoint, from @DaoWen
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Maybe "Added support for retrying a group of jobs"?

File: scheduler/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
...
+- Added support for configurable progress file in Cook Executor, from @shamsimam
+- Added support for configurable environment passed to Cook Executor, from @shamsimam
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Should these two be in a separate changelog for the executor?

File: scheduler/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
...
+### Changed
+- Fixed bug with job group constraints, from @pschorf
+- Fixed bug where Cook Executor jobs were opting in to the heartbeat support, from @shamsimam
+- Changed (simplified) the sandbox directory syncing mechanism for jobs, from @shamsimam
+- Removed jobs from group entity in job-ent->map, from @pschorf
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Did the previous release actually have this issue? I thought it was introduced in an unreleased change

File: scheduler/CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.7.1] - 2017-11-07
```
Comment (dposada) [inline scheduler/CHANGELOG.md]: Shouldn't this be 1.8.0?

File: executor/cook/_version.py (hunk 1 above)
Comment (shamsimam) [reply executor/cook/_version.py]: `devN` is the python equivalent of `SNAPSHOT` in versions. https://www.python.org/dev/peps/pep-0440/#version-scheme

File: scheduler/CHANGELOG.md (hunk 6 above)
Comment (shamsimam) [reply scheduler/CHANGELOG.md]: Changed to `1.8.0`

File: scheduler/CHANGELOG.md (hunk 2 above)
Comment (shamsimam) [reply scheduler/CHANGELOG.md]: Done.

File: scheduler/CHANGELOG.md (hunk 3 above)
Comment (shamsimam) [reply scheduler/CHANGELOG.md]: Changed.

File: scheduler/CHANGELOG.md (hunk 4 above)
Comment (shamsimam) [reply scheduler/CHANGELOG.md]: Just put the last one as it reflects changes in the scheduler.

I'd rather avoid a changelog for the executor for now.

File: scheduler/CHANGELOG.md (hunk 5 above)
Comment (shamsimam) [reply scheduler/CHANGELOG.md]: Spoke with @pschorf, removed this entry.

File: scheduler/CHANGELOG.md (hunk 6 above)
Comment (dposada) [reply scheduler/CHANGELOG.md]: PR title is still 1.7.1

**Comment by shamsimam** [comment]
@DaoWen any additional feedback?

File: scheduler/CHANGELOG.md (hunk 6 above)
Comment (shamsimam) [reply scheduler/CHANGELOG.md]: Changed.

**Pull request merged**